### DISCUSSION TITLE
Deploy on server for web

### DISCUSSION THREAD

**Discussion by Azukiro** [AvaloniaUI/Avalonia#10903] [Q&A]
Hi,

I wanted to try to deploy on a server my Avalonia App for web. There is a possibility for that with dotnet publish or something else ?
Even if it's not ready for production.

**Comment by maxkatz6** [comment]
Not much related to avalonia or even .NET in general. You only need to publish the app with "dotnet publish" which will produce static website, which then can be deployed to the static web site host in any way you prefer.

Microsoft wrote a documentation with multiple choices for Blazor WebAssembly, same can be used here https://learn.microsoft.com/en-us/aspnet/core/blazor/host-and-deploy/webassembly?view=aspnetcore-7.0#hosted-deployment-with-multiple-blazor-webassembly-apps

Or in case of AWS https://aws.amazon.com/getting-started/hands-on/host-static-website/
Or Azure https://learn.microsoft.com/en-us/azure/storage/blobs/storage-blob-static-website
Or GitHub Pages https://blog.flycode.com/how-to-deploy-a-static-website-for-free-using-github-pages
there are way too many options, so you need to do a little research to choose one that suits you.
